include/depthimage_to_laserscan/DepthImageToLaserScan.h

```diff
@@ -187,9 +195,18 @@ namespace depthimage_to_laserscan
       const T* depth_row = reinterpret_cast<const T*>(&depth_msg->data[0]);
       int row_step = depth_msg->step / sizeof(T);
 
-      // offset by certain amonut since the bottom 150 row are mostly floor (noisy data)
-      int offset = (int)(center_y - scan_height/2 + OFFSET_BOTTOM_ROW/2);
-      depth_row += offset*row_step; // Offset to center of image
+      // rows or depth image data between lower/upper bound are used to generated laser scan
+      // offset by certain amonut since the bottom rows are mostly floor (noisy data)
+      int lower_bound = (int)(center_y - scan_height / 2 + offset_bottom_row_ / 2);
+      if(lower_bound <= depth_msg->height / 3){
+        lower_bound = depth_msg->height / 3;
+      }
```

Comment:
The general idea is of this seems correct, but it seems we may be better off using the height_min_ and the height_max_ values to accomplish this instead of pixel offsets from the bottom of the image.  It looks like those values could be used in the use_point method along with the sensor min/max to remove any undesired points.